examples/enqueue_trial.py

```diff
@@ -0,0 +1,39 @@
+"""
+Optuna enqueue_trial example that optimizes a classifier configuration using sklearn.
+
+In this example, we optimize a classifier configuration for Iris dataset. We start a study with
+given parameter values, such as a default and a manually optimized.
```

Comment:
How about replacing `optimized` with `suggested`? IMO, we may usually call optimized parameters if they have been evaluated.
```suggestion
given parameter values, such as a default and a manually suggested one.
```